### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Samju200**
## Task: ` Contact form page for my
contact
 website`
- `add a contact html and the image of the logo `

### CODE REVIEW

**Comment by Samju200** [comment]
> Well done @Samju200
> 
> Please rename your folder from `contactForm` to `HTMLForms`

done

File: submissions/adeyemiJuwon/HTMLforms/contact.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <section>
+      <h1>Contact Us</h1>
+      <p>Address: Martha's Living products Ltd.
+        43 Longley road, Tooting, London, SW17 9LA, United Kingdom</p>
+    <li> Tel number:<a href=" tell:+2348084624301"> +2348084624301</a></li>
+    <br>
+    <li>Email: <a href=" mailto:shopify.raymart7@gmail.com"> shopify.raymart7@gmail.com</a></li>
+    <br>
+    
+  <form action="#" method="POST" enctype="multipart/form-data">
```
Comment (ToluAkin) [inline submissions/adeyemiJuwon/HTMLforms/contact.html:55]: There should be just form tag here. Remove the other elements i.e `action`, `method` and `enctype`

Comment (Samju200) [reply submissions/adeyemiJuwon/HTMLforms/contact.html:55]: okay.
i should create another PR or just push it into my repo only

File: submissions/adeyemiJuwon/HTMLforms/contact.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>contact</title>
+</head>
+<body>
+  <header>
+    <div>
+        <input type="search" name="q" value="" aria-label="Search" placeholder="Search">
```
Comment (ToluAkin) [inline submissions/adeyemiJuwon/HTMLforms/contact.html:11]: @Samju200 Why is the `name` attribute on the input field `q`? Update that.

File: submissions/adeyemiJuwon/HTMLforms/contact.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        43 Longley road, Tooting, London, SW17 9LA, United Kingdom</p>
+    <li> Tel number:<a href=" tell:+2348084624301"> +2348084624301</a></li>
+    <br>
+    <li>Email: <a href=" mailto:shopify.raymart7@gmail.com"> shopify.raymart7@gmail.com</a></li>
+    <br>
+    
+  <form>
+    <h3>Fill this form</h3>
+    <label for="fullname">Full Name</label><br>
+    <input type="text" id="fullname" name="fullname" placeholder="Adeyemi Juwon Samuel" required><br>
```
Comment (ToluAkin) [inline submissions/adeyemiJuwon/HTMLforms/contact.html:58]: Update the label for, input id & name to use camelCase or camel-case i.e `fullName` or `full-name`

File: submissions/adeyemiJuwon/HTMLforms/contact.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    <br>
+    <label for="mail">Email</label><br>
+    <input type="email" id="email" name="email" placeholder="shopify.raymart7@gmail.com" required><br>
+    <br>
+    <label for="phonenumber">Phone Number</label><br>
```
Comment (ToluAkin) [inline submissions/adeyemiJuwon/HTMLforms/contact.html:63]: same as above

File: submissions/adeyemiJuwon/HTMLforms/contact.html (hunk 3 above)
Comment (ToluAkin) [inline submissions/adeyemiJuwon/HTMLforms/contact.html:58]: Do remove the required attribute where added on this form

File: submissions/adeyemiJuwon/HTMLforms/contact.html (hunk 4 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/HTMLforms/contact.html]: > same as above

done

File: submissions/adeyemiJuwon/HTMLforms/contact.html (hunk 3 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/HTMLforms/contact.html]: > Do remove the required attribute where added on this form

done

Comment (Samju200) [reply submissions/adeyemiJuwon/HTMLforms/contact.html]: > Update the label for, input id & name to use camelCase or camel-case i.e `fullName` or `full-name`

done

File: submissions/adeyemiJuwon/HTMLforms/contact.html (hunk 2 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/HTMLforms/contact.html]: > @Samju200 Why is the `name` attribute on the input field `q`? Update that.

Okay
Done

**Pull request merged**